changedetectionio/importer.py

```diff
@@ -182,7 +183,12 @@ def run(self,
                 days, hours = divmod(hours, 24)
                 weeks, days = divmod(days, 7)
                 extras['time_between_check'] = {'weeks': weeks, 'days': days, 'hours': hours, 'minutes': minutes, 'seconds': 0}
-
+            
+            # Openoffice is sometimes converting 'FALSE' to '=FALSE()'
+            if strtobool(str(data.get('dynamic wachet')).replace('=', '').replace('()', '').strip().lower()):
+                extras['fetch_backend'] = 'html_webdriver'
+            else:
+                extras['fetch_backend'] = 'html_requests'
```

Comment:
no, as we discussed, you should not set this if its not set

Reply:
Updated code to explicitly check for boolean and not set it if it is not boolean (or None)